### PR TITLE
atlassian-cli 9.0.0

### PR DESCRIPTION
**Pull request by zachauten**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
@BrewTestBot test this please

File: Formula/atlassian-cli.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class AtlassianCli < Formula
   desc "Command-line interface clients for Atlassian products"
   homepage "https://bobswift.atlassian.net/wiki/pages/viewpage.action?pageId=1966101"
-  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-8.5.0-distribution.zip"
-  sha256 "b9c2d2c9d23852cef34be9efc3813ce4fd2f79fe665c9b9bc784f462749faf89"
+  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-9.0.0-distribution.zip"
+  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```
Comment (chenrui333) [inline Formula/atlassian-cli.rb]: `3bf1760d6d2dae8fad7cfba9666de61ac0f58883c5c2c2744a4a90356670f1a7`

**Comment by chenrui333** [comment]
checksum mismatch issue

File: Formula/atlassian-cli.rb (hunk 1 above)
Comment (zachauten) [reply Formula/atlassian-cli.rb]: Thanks!

**Comment by chenrui333** [comment]
21:44:37 Error: An exception occurred within a child process:
21:44:37   Utils::InreplaceError: inreplace failed
21:44:37 acli.sh:
21:44:37   expected replacement of "`dirname $0`" with #<Pathname:/usr/local/Cellar/atlassian-cli/9.0.0/share>

**Comment by chenrui333** [comment]
line 14, `inreplace cmd + ".sh", "`dirname $0`", share` may be obsolete.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**